src/main/java/core/basesyntax/service/serviceimpl/FruitRawStringParserImpl.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.dto.FruitTransactionDto;
+import core.basesyntax.exception.DataFileCorrupted;
+import core.basesyntax.service.interfaces.FruitRawStringParser;
+import core.basesyntax.service.interfaces.FruitStringsCheck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitRawStringParserImpl implements FruitRawStringParser {
```

Comment:
You are parsing a fruit transaction here, and implementing some parser, so I'd do something like this:
```suggestion
public class FruitTransactionParser implements TransactionParser<FruitTransaction> {
```
and generified the method in the interface to make it return `<T>`, which would make our `TransactionParser` even more handy to reuse in the future